README.md

```diff
@@ -122,6 +123,23 @@ $ ocic pod create --config test/testdata/sandbox_config.json
 ```
 # ocic ctr list
 ```
+#### Recovery from errors
+Information about created pods is stored here `/var/lib/ocid/sandboxes`.
+Currently, this information can get out of sync causing errors such as:
```

Comment:
Do we have a bug specifically for this issue. I recognise that it's going to be a common theme within ocid for quite a while, but we should definitely track it somewhere IMO.